tests/validation_suite/worker_tests/test_worker_update_api.py

```diff
@@ -0,0 +1,144 @@
+# Copyright 2019 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
Don't create the python files per API basis. You can use jrpc wrapper classes to test negative scenarios. Framework should be easy to extend(proxy model)